src/schema/sources.ts

```diff
@@ -649,7 +676,7 @@ export const resolvers: IResolvers<any, Context> = {
     },
     sourceMembers: async (
       _,
-      args: ConnectionArguments & { sourceId: string },
+      { role, ...args }: SourceMemberArgs,
```

Comment:
Why do we only extract role and not sourceId here?
I know it's not related to your change, but maybe we should extract sourceId as well?

Reply:
Makes sense. Let me see if anything fails after destructuring the args and passing it without the `sourceId` to the paginator.